pages/api/availability/eventtype.ts

```diff
@@ -144,6 +144,12 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
         const openingHours = req.body.availability.openingHours || [];
         // const overrides = req.body.availability.dateOverrides || [];
 
+        await prisma.availability.deleteMany({
+          where: {
+            eventTypeId: +req.body.id,
```

Comment:
You need to make sure you are the logged in user too

```suggestion
            eventTypeId: +req.body.id,
            userId: ....
```

Reply:
I don't think you will ever reach this part unless you are logged in... or am I wrong?

Reply:
Indeed! Good catch!